Wash-Ups.md

```diff
@@ -1,13 +1,13 @@
-A wash-up can be held after any significant event that has happened where we believe actions can be taken so that we can better improve process moving forward.  This could be following an expedite issue or at the end of a large project.
+A wash-up can be held after any significant event that has happened where we believe actions can be taken so that we can better improve our performance in the future.  This could be following an expedite issue or at the end of a large project.
 
-A new wash-up meeting template has been created that should help people to prepare for the meeting and ensure that all information on why the meeting is taking place has been filled in and circulated to all relevant parties in advance.
+A new [wash-up meeting template](https://docs.google.com/document/d/1iYaqVGujJ8ARBgKtFsYxXgswjkdnwxB4flneT6m_9uM/edit) has been created to prepare for the meeting. All information on why the meeting is taking place should be filled in and circulated to all relevant parties in advance.
```

Comment:
don't have to change it, but could we remove the temporality of this sentence? Change it to something like

> We have a wash-up meeting template to help prepare for the meeting.

It'll only be a new doc for so long :)